indico/modules/events/operations.py

```diff
@@ -353,3 +373,17 @@ def sort_reviewing_questions(questions, new_positions):
         field.position = index
     db.session.flush()
     logger.info("Reviewing questions of %r reordered by %r", questions[0].event, session.user)
+
+
+def get_event_from_url(url):
+    data = urlparse(url)
+    if not all([data.scheme, data.netloc, data.path]):
+        raise ValueError(_('Invalid event URL'))
+    event_path = re.search('^/event/([0-9]+)/*', data.path)
```

Comment:
I was thinking whether to suggest letting Flask parse the path instead of a regex but IIRC doing that is somewhat messy and we also don't really care *which* endpoint it matches as long as there's an event ID...

So keeping the regex is probably the better option here.